CRM/EventsExtras/Hook/BuildForm/BaseEvent.php

```diff
@@ -58,7 +58,9 @@ protected function hideField(CRM_Event_Form_ManageEvent &$form){
     $settingsValue = SettingsManager::getSettingsValue();
     $hiddenFields = [];
     foreach ($configFields as $config) {
-     if ($settingsValue[$config['name']] == 0 && array_key_exists('css_class', $config['extra_attributes'])){
+     if (isset($settingsValue[$config['name']]) &&
+        $settingsValue[$config['name']] == 0 &&
+        array_key_exists('css_class', $config['extra_attributes'])){
```

Comment:
This could get a bit difficult to comprehend. A better way might be to name the conditionals as variables.
```php
$configNameExists = isset($settingsValue[$config['name']]);
$configNameIsZero = $settingsValue[$config['name']] == 0;
$cssClassExists = array_key_exists('css_class', $config['extra_attributes'];

if ($configNameExists && $configNameIsZero && $cssClassExists {
//code here.
}


```